Makefile

```diff
@@ -19,17 +19,19 @@ USERNAME ?= cosi-project
 REGISTRY_AND_USERNAME ?= $(REGISTRY)/$(USERNAME)
 PROTOBUF_GO_VERSION ?= 1.36.5
 GRPC_GO_VERSION ?= 1.5.1
-GRPC_GATEWAY_VERSION ?= 2.26.1
+GRPC_GATEWAY_VERSION ?= 2.26.3
 VTPROTOBUF_VERSION ?= 0.6.0
-GOIMPORTS_VERSION ?= 0.30.0
+GOIMPORTS_VERSION ?= 0.31.0
+GOMOCK_VERSION ?= 0.5.0
 DEEPCOPY_VERSION ?= v0.5.6
-GOLANGCILINT_VERSION ?= v1.64.5
+GOLANGCILINT_VERSION ?= v1.64.6
 GOFUMPT_VERSION ?= v0.7.0
-GO_VERSION ?= 1.24.0
+GO_VERSION ?= 1.24.1
```

Comment:
I think it makes sense to first update `kres` for new Go & golangci-lint

Reply:
You mean 1.24.2 right?